### PR TITLE
Prepare unit tests for more commits

### PR DESCRIPTION
**Pull request by jennyf19**
- Add a test constants class
- Fix failing unit test which is fixed now that we get client_info, so we no longer fail if there is no tid in the jwt, we pull it from the claim
- I've only added on b2c authority test, but otherwise have not changed them

### CODE REVIEW

File: tests/Microsoft.Identity.Web.Test/TestConstants.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+{
+    internal static class TestConstants
+    {
+        public const string ProductionPrefNetworkEnvironment = "login.microsoftonline.com";
+        public const string ProductionPrefNetworkUSEnvironment = "login.microsoftonline.us";
+        public const string ProductionNotPrefEnvironmentAlias = "sts.windows.net";
+
+        public const string LocalHost = "https://localhost";
+        
+        public const string ApiAudience = "api://1EE5A092-0DFD-42B6-88E5-C517C0141321";
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Web.Test/TestConstants.cs]: Wouldn't you want to have "api://" + ApiClientId  ?

File: tests/Microsoft.Identity.Web.Test/TestConstants.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        public const string B2CAuthorityWithV2 = B2CAuthority + "/v2.0";
+        public const string B2CCustomDomainAuthority = B2CCustomDomainInstance + "/" + B2CTenant + "/" + B2CSuSiUserFlow;
+        public const string B2CCustomDomainAuthorityWithV2 = B2CCustomDomainAuthority + "/v2.0";
+
+        public const string B2CIssuer = B2CInstance + "/" + B2CTenantAsGuid + "/v2.0";       
+        public const string B2CIssuer2 = B2CInstance2 + "/" + B2CTenantAsGuid + "/v2.0";       
+        public const string B2CCustomDomainIssuer = B2CCustomDomainInstance + "/" + B2CTenantAsGuid + "/v2.0";        
+
+        //Claims
+        public const string Tid = "tid";
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Web.Test/TestConstants.cs]: ClaimNameTid ?

File: tests/Microsoft.Identity.Web.Test/TestConstants.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        public const string Iss = "iss";
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Web.Test/TestConstants.cs]: ClaimNameIss?

File: tests/Microsoft.Identity.Web.Test/TestConstants.cs [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        public const string Tfp = "tfp"; //Trust Framework Policy for B2C (aka userflow/policy)
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Web.Test/TestConstants.cs]: ClaimNameTfp?

File: tests/Microsoft.Identity.Web.Test/TestConstants.cs [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+namespace Microsoft.Identity.Web.Test
+{
+    internal static class TestConstants
+    {
+        public const string ProductionPrefNetworkEnvironment = "login.microsoftonline.com";
+        public const string ProductionPrefNetworkUSEnvironment = "login.microsoftonline.us";
+        public const string ProductionNotPrefEnvironmentAlias = "sts.windows.net";
+
+        public const string LocalHost = "https://localhost";
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Web.Test/TestConstants.cs]: HttpLocalhost?

File: tests/Microsoft.Identity.Web.Test/TestConstants.cs (hunk 1 above)
Comment (jennyf19) [reply tests/Microsoft.Identity.Web.Test/TestConstants.cs]: yes. thanks. will fix.

---
In reply to: [381455089](https://github.com/AzureAD/microsoft-identity-web/pull/22#discussion_r381455089) [](ancestors = 381455089)

**Comment by jennyf19** [comment]
yes. great suggestion. thanks

---
In reply to: [361322935](https://github.com/AzureAD/microsoft-identity-web/pull/22#issuecomment-361322935) [](ancestors = 361322935)

**Pull request merged**